packages/web/src/components/premium-content-purchase-modal/components/PurchaseContentFormFields.tsx

```diff
@@ -56,33 +36,23 @@ export const PurchaseContentFormFields = ({
   stage,
   isUnlocking
 }: PurchaseContentFormFieldsProps) => {
-  const mobile = isMobile()
   const payExtraAmountPresetValues = usePayExtraPresets()
   const [{ value: purchaseMethod }, , { setValue: setPurchaseMethod }] =
     useField(PURCHASE_METHOD)
   const isPurchased = stage === PurchaseContentStage.FINISH
 
   const { data: balanceBN } = useUSDCBalance()
```

Comment:
any reason why we donot poll here? asking because we poll on mobile.

Reply:
@dharit-tan maybe you know?

Reply:
i think we don't necessarily need to poll here. i went through and deduped all the places we poll and decided that we only really need to poll once the user clicks through to the add funds part of the flow. lmk if you disagree.
easy way to check is just grep `isPolling` in the codebase. can also add a log inside of the refresh function in `useUSDCBalance` to see if there are multiple stacked pollers.